lib/new_relic/control/frameworks/rails.rb

```diff
@@ -10,6 +10,9 @@ module Frameworks
       # Rails specific configuration, instrumentation, environment values,
       # etc.
       class Rails < NewRelic::Control::Frameworks::Ruby
+        INSTALLED_SINGLETON = NewRelic::Agent.config
+        INSTALLED = :@browser_monitoring_installed
```

Comment:
I really like this refactor. Would you be open to updating the `INSTALLED` constant name to include "browser" in some way? When I was reading the tests, I got a bit confused about what specifically was being installed.

I think `INSTALLED_SINGLETON` is fine as-is, but could be `NEW_RELIC_AGENT_CONFIG` if you wanted to have more specificity there as well.

Reply:
Updated the names with [7ca493f](https://github.com/newrelic/newrelic-ruby-agent/pull/2248/commits/7ca493f627d67de4dbf57b1a3f8173995f0a1f59). I'd prefer to keep the "AGENT_CONFIG" part out of the name itself for flexibility. When I first created the constant I envisioned the control class not being particularly concerned with what class is being used for singleton work - just the class's name. By keeping the singleton's name out of the constant name, a different value could be used in future.